adoc/Release-Notes.adoc

```diff
@@ -33,20 +33,122 @@ endif::[]
 [id='sec.major-change']
 == Major Changes
 
+[id='sec.1_5_2']
+=== Release 1.5.2, February 2020
+
+[id='sec.1_5_2.new']
+==== What Is New?
+* {scf} has been updated to version 2.20.3:
+** cf-deployment has been updated to 12.17
+** Optimized startup time for various roles
+** Added/fixed podAntiAffinity rules for various roles
+* The Stratos UI has been updated to version 2.7.0:
+** Added support for {caasp} V4.1 and {k8s} 1.16+ environments
```

Comment:
Stratos already supports Kube 1.16+ prior to 2.7.0 - suggest you replace content with:

* The Stratos UI has been updated to version 2.7.0:
** Improvements to the Kubernetes Dashboard integration (improved configuration and reliability)
** Improved support for synchronous creation of service instances
** Branding updated to "SUSE Stratos Console"

Reply:
Thank you sir for the corrected info.